src/Kris/LaravelFormBuilder/Form.php

```diff
@@ -1039,6 +1045,8 @@ public function validate($validationRules = [], $messages = [])
         $this->validator = $this->validatorFactory->make($this->getRequest()->all(), $rules, $messages);
         $this->validator->setAttributeNames($fieldRules['attributes']);
 
+        \Event::fire(new BeforeFormValidation($this, $this->validator));
```

Comment:
Do not use Laravel facades in package. Inject all dependencies through constructor.